### PR TITLE
Storybook for ZeroHeight

### PR DESCRIPTION
**Pull request by GuillaumeNury**
## Description
* Bumped dependencies
* Added zero height mode for stories

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1543

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1543

**Review by bastienguillon — DISMISSED** [review]

File: .storybook/main.js [hunk 1]
```diff
@@ -1,30 +1,17 @@
...
     enableIvy: true,
   },
-  webpackFinal: async (config) => {
-    config.resolve.plugins = [
-      new TsconfigPathsPlugin({
-        configFile: path.resolve(__dirname, '../tsconfig.json'),
-      }),
-    ];
-    return config;
-  },
```
Comment (shajz) [inline .storybook/main.js:29]: We don't need this anymore? Hell yeah

File: package.json [hunk 2]
```diff
@@ -87,15 +84,15 @@
...
-		"stylelint": "^13.11.0",
-		"ts-node": "~10.0.0",
+		"jest": "^27.5.1",
+		"jest-preset-angular": "^11.1.1",
+		"ng-packagr": "^13.2.1",
+		"postcss": "^8.4.6",
+		"sass": "^1.49.8",
+		"stylelint": "^14.5.3",
+		"ts-node": "~10.5.0",
 		"tsconfig-paths-webpack-plugin": "^3.5.2",
```
Comment (shajz) [inline package.json:94]: We can remove this :)

File: package.json [hunk 3]
```diff
@@ -35,48 +35,45 @@
...
-		"@angular/core": "13.0.2",
-		"@angular/forms": "13.0.2",
-		"@angular/platform-browser": "13.0.2",
-		"@angular/platform-browser-dynamic": "13.0.2",
-		"@angular/router": "13.0.2",
-		"@ngx-formly/core": "^5.10.14",
-		"cpy": "^8.1.1",
+		"@angular/animations": "13.2.4",
+		"@angular/cdk": "^13.2.4",
+		"@angular/common": "13.2.4",
```
Comment (shajz) [inline package.json:40]: Should we bump some peer deps in each lib's package.json?

File: .storybook/main.js (hunk 1 above)
Comment (GuillaumeNury) [reply .storybook/main.js:29]: Nope, it's working without this

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1543

**Review by shajz — APPROVED** [review]

**Review by jeremie-lucca — APPROVED** [review]

**Pull request merged**